src/app/components/layout/Footer/Footer.tsx

```
@@ -58,7 +58,8 @@ const Footer = async ({ lng }: PropsWithLng) => {
             <div className="flex flex-col gap-2">
               {links.map(({ link, name, key }: FooterLink) =>
                 key === 'bugReport' ? (
-                  <CSLink key={name}>{name}</CSLink>
+                  // <CSLink key={name}>{name}</CSLink>
+                  <></>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)